core/sdk-core/src/main/java/software/amazon/awssdk/core/sync/RequestBody.java

```diff
@@ -70,6 +73,13 @@ public long contentLength() {
         return contentLength;
     }
```

Comment:
Should we deprecate this method?

Reply:
Yea, we can also do that. But it's going to break the customers that are using this method currently. I can do the change and bump up the minor version though.

Reply:
I meant adding `@deprecate` annotation, so it's not a breaking change.

Reply:
Got you, will do.